### PR TITLE
[Mistweaver] Add enveloping breath to chiji stat box

### PR DESCRIPTION
**Pull request by pingypong**
updated chiji stat box to include healing from enveloping breath, similar to yulon stat box

### CODE REVIEW

**Comment by abelito75** [comment]
Good fix. I didn't think chi-ji was gonna get envb back (till last build he didn't have it)

File: src/parser/monk/mistweaver/modules/talents/InvokeChiJi.js [hunk 1]
```diff
@@ -7,29 +7,53 @@ import Events from 'parser/core/Events';
...
   statistic() {
     return (
-      <StatisticBox
-        position={STATISTIC_ORDER.OPTIONAL(50)}
-        icon={<SpellIcon id={SPELLS.INVOKE_CHIJI_THE_RED_CRANE_TALENT.id} />}
-        value={<ItemHealingDone amount={this.healing} />}
-        label="Chi-Ji Healing"
-      />
+      <>
+        <StatisticBox
```
Comment (abelito75) [inline src/parser/monk/mistweaver/modules/talents/InvokeChiJi.js:36]: We have deprecated StatisticBox and moved to Statistic as our new layout for these things!

**Review by abelito75 — CHANGES_REQUESTED** [review]
Could you send a screenshot of what this looks like now?

File: src/parser/monk/mistweaver/modules/talents/InvokeChiJi.js (hunk 1 above)
Comment (pingypong) [reply src/parser/monk/mistweaver/modules/talents/InvokeChiJi.js]: image
updated

**Comment by abelito75** [comment]
If this is your first contribution you're gonna need to add yourself to the contributors and then import yourself into the change log :) 
After that I think we are all good

**Comment by pingypong** [comment]
added. let me know if you see anything else

**Review by abelito75 — APPROVED** [review]
Looks good now

**Comment by abelito75** [comment]
Let me take one more gander at this before merging it

**Pull request merged**